tests/osaka/eip7883_modexp_gas_increase/test_eip_mainnet.py

```diff
@@ -0,0 +1,63 @@
+"""
+EIP-7883 ModExp gas cost increase tests.
+
+Tests for ModExp gas cost increase in
+[EIP-7883: ModExp Gas Cost Increase](https://eips.ethereum.org/EIPS/eip-7883).
+"""
+
+from typing import Dict
+
+import pytest
+from execution_testing import (
+    Alloc,
+    StateTestFiller,
+    Transaction,
+)
+
+from ...byzantium.eip198_modexp_precompile.helpers import ModExpInput
+from .spec import ref_spec_7883
+
+REFERENCE_SPEC_GIT_PATH = ref_spec_7883.git_path
+REFERENCE_SPEC_VERSION = ref_spec_7883.version
+
+
+@pytest.mark.parametrize(
+    "modexp_input,modexp_expected",
+    [
+        pytest.param(
+            ModExpInput(
+                base="ff" * 31 + "fc",
+                exponent="02",
+                modulus="05",
+                declared_base_length=32,
+                declared_exponent_length=1,
+                declared_modulus_length=1,
+            ),
+            bytes([4]),
+            id="32-bytes-long-base",
+        ),
+        pytest.param(
+            ModExpInput(
+                base="ff" * 32 + "fb",
+                exponent="02",
+                modulus="05",
+                declared_base_length=33,
+                declared_exponent_length=1,
+                declared_modulus_length=1,
+            ),
+            bytes([1]),
+            id="33-bytes-long-base",  # higher cost than 32 bytes
+        ),
+    ],
+)
+@pytest.mark.valid_at("Osaka")
```

Comment:
We should mark these tests as mainnet only like we do in: https://github.com/ethereum/execution-spec-tests/pull/1511/files#diff-a9add5727efb38c476ff59942c2a0732eb1f9dde4958dd3b573a36fe3444bf05R15
```suggestion
@pytest.mark.valid_at("Osaka")
@pytest.mark.mainnet
```

We only want to fill/execute these specifically for mainnet validation.